src/app/child-dev-project/children/child-details/child-details.component.html

```diff
@@ -69,14 +69,25 @@ <h1 class="page-header section-child">
           <mat-form-field style='width: 300px;'>
             <input matInput formControlName="name" placeholder="Name" title="name" type="text">
           </mat-form-field>
+          <div *ngIf="this.form.get('name').invalid && validateForm" class="validationWarning">
+                Name is required
+          </div>
 
-          <mat-form-field style='width: 100px;'>
+          <div>
+          <mat-form-field>
             <input matInput formControlName="projectNumber" placeholder="Project Number" type="text">
           </mat-form-field>
+          </div>
 
+          <div>
           <mat-form-field>
             <input matInput formControlName="center" placeholder="Center" type="text">
           </mat-form-field>
+          <div *ngIf="this.form.get('center').invalid && validateForm" class="validationWarning">
+            Center is required
+          </div>
```

Comment:
when looking into the styling to move the error message a little closer to the field, I found the documentation for mat-error: https://material.angular.io/components/form-field/overview#error-messages
This is probably what we would want to use instead of our own div. Or do you have a specific reason to do it this way?

mat-error would also automatically use an optimized styling:
![image](https://user-images.githubusercontent.com/1682541/72552921-1400c280-3898-11ea-8fee-061002c761e7.png)

Reply:
i implemented mat-error and it worked fine